src/build-tools/common-grunt-rules.ts

```diff
@@ -95,6 +96,21 @@ export class Rule {
     }
   }
 
+  public jasmineSingleSpec(file :string) :JasmineRule {
```

Comment:
I didn't want to do a name conflict with the current rule since the behaviour changed